pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -28,9 +44,18 @@ func (r *ReconcileStorageCluster) ensureStorageClasses(instance *ocsv1.StorageCl
 	if err != nil {
 		return err
 	}
+	err = r.createStorageClasses(scs, reqLogger)
+	if err != nil {
+		return err
```

Comment:
name your error

Reply:
`createStorageClasses()` should be responsible for elaborating on its own errors, we just need to pass it up. This is outside the scope of this PR.